SteamKit2/SteamKit2/Steam/WebAPI/WebAPI.cs

```diff
@@ -222,68 +222,61 @@ public async Task<KeyValue> CallAsync( HttpMethod method, string func, int versi
                     throw new ArgumentNullException( nameof(func) );
                 }
 
-                if ( args == null )
+                if ( args != null && args.TryGetValue( "format", out var format ) )
                 {
-                    args = new Dictionary<string, object>();
+                    if ( !( format is string formatText ) || formatText != "vdf" )
+                    {
+                        throw new ArgumentException( $"{nameof( args )} include unsupported {nameof( format )}: {format}" );
```

Comment:
Printing {format} seems sketchy when it can be a non string. Perhaps just reword this exception to say that only format=vdf is supported?